builtins/amp-img.js

```diff
@@ -106,6 +135,10 @@ export class AmpImg extends BaseElement {
       if (!IS_ESM) {
         guaranteeSrcForSrcsetUnsupportedBrowsers(this.img_);
       }
+
+      if (AmpImg.V1() && !this.img_.complete) {
+        this.onReadyState(ReadyState.LOADING);
```

Comment:
nonblocking nit: I explained the logic more fully in the design doc, but I think a slightly better `setReadyState`

Reply:
Done